community/civilized-discussion.md

```diff
@@ -0,0 +1,112 @@
+
+Some examples:
+- A discussion exploring what the tradeoffs would be between using C# vs F# could be a positive and educational experience. However, a comment saying “C# is rubbish because …” is not helpful, does not add to the educational experience and would be deleted.
+- Any conversations around topics such as politics or religion are not appropriate in this space. Neither are topics that are fought with a religious fervor (e.g. whether VIM or Emacs is better).
+- Any sort of comment that implies “noobs do …” or “real developers use …” will be deleted.
+
+<a name="improve"></a>
+
+## [Improve the Discussion](#improve)
+
+Help us make this a great place for discussion by **always adding something positive to the discussion**. If you are not sure your post adds to the conversation, think over what you want to say and try again later.
+
+One way to improve the discussion is by discovering ones that are already happening. Spend time browsing the topics here before replying or starting your own, and you’ll have a better chance of meeting others who share your interests.
+
+The topics discussed here matter to us, and we want you to act as if they matter to you, too. Be respectful of the topics and the people discussing them, even if you disagree with some of what is being said.
+
+<a name="agreeable"></a>
+
+## [Be Agreeable, Even When You Disagree](#agreeable)
+
+You may wish to respond by disagreeing. That’s fine. But remember to _criticize ideas, not people_. Please avoid:
+
+* Name-calling
+* Ad hominem attacks
+* Responding to a post’s tone instead of its actual content
+* Knee-jerk contradiction
+
+Instead, provide thoughtful insights that improve the conversation.
+
+Write your opinions **as opinions**. Rather than saying "everyone knows Python is the best language for beginners" or "Python is the best language for beginners", say "I feel that Python is the best language for beginners because ...". We take this seriously as it's an innocent looking habit that can be extremely disruptive to an online community. If you constantly write opinions as facts, we will ask you to stop and refer you to these guidelines. If you continue, we may choose to ban you.
+
+<a name="derailing"></a>
+
+## [Stay on topic - don’t derail](#derail)
+
+When posting, always think about whether you’re moving the conversation in the direction that the original poster intended, or whether you’re pulling it in a different direction. If it’s the latter, it's generally better to create a new topic.
+
+For example, if there’s a topic on cats, but you prefer dogs, don’t create a post saying “cats are rubbish - I like dogs” - instead create a new topic on dogs with your favorite puppy pictures in.
```

Comment:
I'd like to report a violation: http://forum.exercism.org/t/jeremy-prefers-dogs-in-a-post-about-cats/72/2

Reply:
Yes - this was deliberate :) But you're note I was positive "I don't normally like dogs but...". However, we did raise an internal issue about @lorettaexercism's behaviour in that thread ;)